public/app/features/alerting/unified/components/receivers/ReceiversTable.tsx

```diff
@@ -203,8 +203,8 @@ function NotifiersTable({ notifiersState }: NotifiersTableProps) {
       {
         id: 'lastNotifyDuration',
         label: 'Last duration',
-        renderCell: ({ data: { lastNotifyDuration } }) => (
-          <>{durationIsNull(lastNotifyDuration) ? '-' : lastNotifyDuration}</>
+        renderCell: ({ data: { lastNotify, lastNotifyDuration } }) => (
+          <>{isLastNotifyNullDate(lastNotify) && durationIsNull(lastNotifyDuration) ? '-' : lastNotifyDuration}</>
```

Comment:
I understand from this change that we consider `lastNotifyDuration` as null only when we have a null `lastNotify`, right?. If it's supposed to be this way, LGTM 🎉

Reply:
Exactly! As I said in the description, we're rounding down durations, so if the attempt takes less than 1ms the value will be `0s`, and it's a totally valid one.